metroplex.js

```diff
@@ -212,6 +177,18 @@ Metroplex.readable('servers', function servers(self, fn) {
 });
 
 /**
+ * Reset the time to live for a registered spark.
+ *
+ * @param {Spark} spark The connection/spark from Primus.
+ * @returns {Metroplex}
+ * @api private
+**/
+Metroplex.readable('heartbeat', function heartbeat(spark) {
+  this.redis.psetex(this.namespace + 'spark:' + spark.id, this.sparkTimeout, this.address);
```

Comment:
We only need to update the timeout of the key, so can use `PEXPIRE`, without setting the value again.